src/StaticFAQ/FAQCategoryList.js

```diff
@@ -182,6 +182,10 @@ class RawFAQCategoryList extends React.Component<Props> {
       return <StaticFAQError />;
     }
 
+    const FAQCategory = idx(rendererProps.props, _ => _.FAQCategory);
+
+    if (FAQCategory === null) return <div>Category not found</div>;
```

Comment:
I think rendering the error for FAQs would be better: `src/StaticFAQ/StaticFAQError.js`